plugins/block_storage/lib/block_storage/driver/block_storage_driver.rb

```diff
@@ -69,6 +71,23 @@ def test(filter={})
         puts "test"
       end
 
+      def volume_action(id,status={})
```

Comment:
'action' is more generic than just for updating the status, maybe rename it to make future extension easier